### PR TITLE
dnsproxy: init at 0.20.0

### PR DESCRIPTION
**Pull request by contrun**
Init [dnsproxy](https://github.com/AdguardTeam/dnsproxy)

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

**Comment by risicle** [comment]
Seems to WFM, macos 10.13

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -1365,6 +1365,12 @@
     githubId = 5684605;
     name = "Cole Scott";
   };
+  contrun = {
```
Comment (Lassulus) [inline maintainers/maintainer-list.nix]: can you make a separate commit out of the maintainers addition?

Comment (contrun) [reply maintainers/maintainer-list.nix]: Done

**Review by Lassulus — APPROVED** [review]
tested with nix-review, started the dnsproxy and threw a query against it
